lang/csharp/src/apache/main/Avro.main.csproj

```diff
@@ -25,8 +25,11 @@
     <PackageId>Confluent.Apache.Avro</PackageId>
     <Title>Confluent.Apache.Avro</Title>
     <AssemblyName>Confluent.Apache.Avro</AssemblyName>
-    <VersionPrefix>1.7.7.6</VersionPrefix>
+    <VersionPrefix>1.7.7.7</VersionPrefix>
     <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>false</DelaySign>
+    <AssemblyOriginatorKeyFile>Confluent.Apache.Avro.snk</AssemblyOriginatorKeyFile>
```

Comment:
Is this the public key?
Just want to make sure we don't include the private signing key in the repo.

Reply:
this does indeed contain the private key, however that is what we want. see: https://stackoverflow.com/questions/36141302/why-is-it-recommended-to-include-the-private-key-used-for-assembly-signing-in-op